components/coachindicator/index.css

```diff
@@ -0,0 +1,99 @@
+/*!
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import "animation.css";
+
+.spectrum-CoachIndicator {
+  --spectrum-coach-indicator-ring-border-size: var(--spectrum-border-width-200);
+  --spectrum-coach-indicator-min-inline-size: calc(var(--spectrum-coach-indicator-ring-diameter) * 3);
+  --spectrum-coach-indicator-min-block-size: calc(var(--spectrum-coach-indicator-ring-diameter) * 3);
+  --spectrum-coach-indicator-ring-inline-size: var(--spectrum-coach-indicator-ring-diameter);
+  --spectrum-coach-indicator-ring-block-size: var(--spectrum-coach-indicator-ring-diameter);
+
+  --spectrum-coach-indicator-top: calc(var(--spectrum-coach-indicator-ring-diameter) * 0.75);
+  --spectrum-coach-indicator-left: calc(var(--spectrum-coach-indicator-ring-diameter) * 0.75);
+
+  /* animation */
+  --spectrum-coach-animation-indicator-ring-duration: 3000ms;
+  --spectrum-coach-animation-indicator-ring-inner-delay-multiple: -0.5;
+  --spectrum-coach-animation-indicator-ring-center-delay-multiple: -0.66;
+  --spectrum-coach-animation-indicator-ring-outer-delay-multiple: -1;
+  --spectrum-coach-indicator-quiet-animation-ring-inner-delay-multiple: -0.33;
+  --spectrum-coach-indicator-animation-name: pulse;
+  --spectrum-coach-indicator-inner-animation-delay-multiple: var(--spectrum-coach-animation-indicator-ring-inner-delay-multiple);
+
+  &--quiet {
+    --mod-coach-indicator-min-inline-size: calc(var(--mod-coach-indicator-quiet-ring-diameter, var(--spectrum-coach-indicator-quiet-ring-diameter)) * 2.75);
+    --mod-coach-indicator-min-block-size: calc(var(--mod-coach-indicator-quiet-ring-diameter, var(--spectrum-coach-indicator-quiet-ring-diameter)) * 2.75);
+    --mod-coach-indicator-ring-inline-size: var(--mod-coach-indicator-quiet-ring-diameter, var(--spectrum-coach-indicator-quiet-ring-diameter));
+    --mod-coach-indicator-ring-block-size: var(--mod-coach-indicator-quiet-ring-diameter, var(--spectrum-coach-indicator-quiet-ring-diameter));
+
+    --mod-coach-indicator-top: calc(var(--spectrum-coach-indicator-quiet-ring-diameter) * 0.75);
+    --mod-coach-indicator-left: calc(var(--spectrum-coach-indicator-quiet-ring-diameter) * 0.75);
+    --mod-coach-indicator-animation-name: pulse--quiet;
+    --mod-coach-indicator-inner-animation-delay-multiple: var(--mod-coach-indicator-quiet-animation-ring-inner-delay-multiple:, var(--spectrum-coach-indicator-quiet-animation-ring-inner-delay-multiple:));
```

Comment:
There is an extra Colon in coachindicator.css after --mod-coach-indicator-quiet-animation-ring-inner-delay-multiple:

Reply:
Fixed! Thanks for catching this.